src/main/java/to/us/resume_builder/ApplicationConfiguration.java

```diff
@@ -0,0 +1,83 @@
+package to.us.resume_builder;
+
+import com.google.gson.Gson;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class enables the use of a configuration file with the program.
+ */
+public class ApplicationConfiguration {
+    private static ApplicationConfiguration instance = null;
+    private Map<String, Object> configuration;
+
+    /**
+     * Construct the <code>ApplicationConfiguration</code> singleton instance.
+     */
+    public ApplicationConfiguration() {
```

Comment:
Just a suggestion: before reading in the configurations, maybe we should have it check whether there is a config file and create if there is none [File f = new File(CONFIG); if (!f.exists()) createConfig();]

Reply:
Good idea. Done.